query/control/controller_test.go

```diff
@@ -270,7 +265,7 @@ func TestController_QueryQueueError(t *testing.T) {
 	}
 
 	// Third "normal" query
-	q, err := ctrl.Query(context.Background(), makeRequest(mockCompiler))
+	q, err := ctrl.Query(context.Background(), (mockCompiler))
```

Comment:
Nitpick: the parens around `mockCompiler` aren't needed (I think), same for lines below

Reply:
Happily, `gofmt` has an option to take out useless parens. Fixed.